tests/integration/checkpoint.bats

```diff
@@ -144,6 +144,35 @@ function simple_cr() {
 	simple_cr
 }
 
+@test "checkpoint --pre-dump (with invalid parent path)" {
+
+	setup_pipes
+	runc_run_with_pipes test_busybox
+
+	mkdir parent-dir
+	runc --criu "$CRIU" checkpoint --pre-dump --image-path ./parent-dir test_busybox
+	[ "$status" -eq 0 ]
+
+	# busybox should still be running
+	testcontainer test_busybox running
+
+	# checkpoint the running container
+	mkdir image-dir
+	mkdir work-dir
+
+	# runc should  fail with absolute parent image path.
+	runc --criu "$CRIU" checkpoint --parent-path "$(pwd)"/parent-dir --work-path ./work-dir --image-path ./image-dir test_busybox
+	[[ "${output}" == *"--parent-path"* ]]
+	[ "$status" -ne 0 ]
+
+	# runc should fail with invalid parent image path.
+	runc --criu "$CRIU" checkpoint --parent-path ./parent-dir --work-path ./work-dir --image-path ./image-dir test_busybox
+	[[ "${output}" == *"--parent-path"* ]]
+	[ "$status" -ne 0 ]
+
+	check_pipes
```

Comment:
As per the above comment, remove this. This is to check that the restored container works, we don't restore the container so ...